docs/operations/move.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Move"
+linkTitle: "Move"
+---
+
+## Moving Ondat volume instances
+
+A volume instance (replica or master) can be moved freely between nodes, this can be extremely useful in very specific scenarios, and it's available through a new command in the command-line tool.
```

Comment:
Same point about "primary" here.

It's also good to mention the specific scenario this is useful for. I suspect the main use case is to re-balance a cluster, reducing load on over-loaded nodes

Reply:
Fixed the wording and added a paragraph about that.